Client/Frontend/Browser/ScreenshotHelper.swift

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/**
+ * Handles screenshots for a given browser, including pages with non-webview content.
+ */
+protocol ScreenshotHelper {
+    func takeScreenshot(tab: Browser, aspectRatio: Float?, quality: Float) -> UIImage?
```

Comment:
A `nil` aspect ratio is weird and error prone. Why not `aspectRation: Float = 1.0` instead? It is better to choose a good default then to allow nil values imo. Specially for value types like floats.